src/Twig/Extension/ArrayExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Bolt\Twig\Extension;
+
+use Bolt\Twig\Runtime;
+use Twig_Extension as Extension;
+
+/**
+ * Array functionality Twig extension.
+ *
+ * @internal
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class ArrayExtension extends Extension
```

Comment:
There's still no reason to have a separate runtime for this class because it doesn't have any dependencies